EngineLayer/Analysis/AnalysisEngine.cs

```diff
@@ -588,10 +695,14 @@ protected override MyResults RunSpecific()
                             writeHistogramPeaksAction(myTreeStructure, searchModes[j].FileNameAddition);
                         }
                     }
+
                     else
                     {
                         Status("Running FDR analysis on unique peptides...", nestedIds);
-                        writePsmsAction(DoFalseDiscoveryRateAnalysis(orderedPsmsWithPeptides.GroupBy(b => b.FullSequence).Select(b => b.FirstOrDefault()), searchModes[j]), "uniquePeptides" + searchModes[j].FileNameAddition);
+                        if (quantify)
+                            writePsmsAction(DoFalseDiscoveryRateAnalysis(orderedPsmsWithPeptides.GroupBy(b => b.FullSequence).Select(b => b.ToList().OrderByDescending(p => p.newPsm.apexIntensity).FirstOrDefault()), searchModes[j]), "uniquePeptides" + searchModes[j].FileNameAddition);
```

Comment:
No need for sorting by apex intensity. 
Revert to previous single writePsmsAction